---
title: 'Genomic exploration of individual giant ocean viruses'
authors:
  - William H Wilson
  - Ilana C Gilg
  - admin
  - Erin K Field
  - Sergey Koren
  - Gary R LeCleir
  - Joaquín Martínez Martínez
  - Nicole J Poulton
  - Brandon K Swan
  - Ramunas Stepanauskas
  - Steven W Wilhelm
date: '2017-08-01T00:00:00Z'
doi: ''

# Schedule page publish date (NOT publication's date).
publishDate: '2017-01-01T00:00:00Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ['2']

# Publication name and optional abbreviated publication name.
publication: '*The ISME journal, 11*(8)'
publication_short: ''

abstract: 'Viruses are major pathogens in all biological systems. Virus propagation and downstream analysis remains a challenge, particularly in the ocean where the majority of their microbial hosts remain recalcitrant to current culturing techniques. We used a cultivation-independent approach to isolate and sequence individual viruses. The protocol uses high-speed fluorescence-activated virus sorting flow cytometry, multiple displacement amplification (MDA), and downstream genomic sequencing. We focused on ‘giant viruses’ that are readily distinguishable by flow cytometry. From a single-milliliter sample of seawater collected from off the dock at Boothbay Harbor, ME, USA, we sorted almost 700 single virus particles, and subsequently focused on a detailed genome analysis of 12. A wide diversity of viruses was identified that included Iridoviridae, extended Mimiviridae and even a taxonomically novel (unresolved) giant virus. We discovered a viral metacaspase homolog in one of our sorted virus particles and discussed its implications in rewiring host metabolism to enhance infection. In addition, we demonstrated that viral metacaspases are widespread in the ocean. We also discovered a virus that contains both a reverse transcriptase and a transposase; although highly speculative, we suggest such a genetic complement would potentially allow this virus to exploit a latency propagation mechanism. Application of single virus genomics provides a powerful opportunity to circumvent cultivation of viruses, moving directly to genomic investigation of naturally occurring viruses, with the assurance that the sequence data is virus-specific, non-chimeric and contains no cellular contamination.'

# Summary. An optional shortened abstract.
summary:
tags:

featured: false

# links:
# - name: ""
#   url: ""
url_pdf: https://www.nature.com/articles/ismej201761.pdf


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

---


